src/bundle.c

```diff
@@ -760,8 +775,6 @@ static enum swupd_code install_bundles(struct list *bundles, struct list **subs,
 		if (is_tracked_bundle(bundle)) {
 			fprintf(stderr, "Warning: Bundle \"%s\" is already installed, skipping it...\n", bundle);
 			already_installed++;
-			/* track as installed since they tried to install it */
```

Comment:
Not sure if we should remove this line. And we need better function names.
is_tracked_bundle() is checking if bundle is installed, and not if it's tracked.

Reply:
Ah indeed, I was fooled by the naming X(.